demux/utils/novaseq_samplesheet.py

```diff
@@ -0,0 +1,181 @@
+""" Create a samplesheet for Novaseq flowcells """
+
+import csv
+
+from .runparameters import NovaseqRunParameters
+from .samplesheet import Samplesheet
+
+
+class CreateNovaseqSamplesheet:
+    """ Create a raw sample sheet for Novaseq flowcells """
+
+    LIMS_KEYS = [
+        "fcid",
+        "lane",
+        "sample_id",
+        "sample_ref",
+        "index",
+        "index2",
+        "sample_name",
+        "control",
+        "recipe",
+        "operator",
+        "project",
+    ]
+
+    def __init__(
+        self, flowcell, indexlength, pad, raw_samplesheet, dummy_indexes, runs_dir
+    ):
+        self.flowcell = flowcell
+        self.indexlength = indexlength
+        self.pad = pad
+        self.raw_samplesheet = raw_samplesheet
+        self.dummy_indexes = dummy_indexes
+        self.runparameters = NovaseqRunParameters(self.flowcell, runs_dir)
+
+    @property
+    def header(self):
+        """ Create the sample sheet header """
+        return list(Samplesheet.header_map.values())
+
+    @staticmethod
+    def get_project(project):
```

Comment:
```suggestion
    def get_project_name(project):
```
Or what does the first part represent?